README.md

```diff
@@ -1,72 +1,93 @@
-# io-onboarding-pa
-This repository contains the code of the backend used by the onboarding portal for public administrations of the IO project.
+# IO PA Onboarding backend (API)
 
-## How to run the application
+This repository contains the code of the backend application of the IO Public Administrations onboarding portal.
+
+## What is IO?
+
+More informations about the IO can be found on the [Digital Transformation Team website](https://teamdigitale.governo.it/en/projects/digital-citizenship.htm)
```

Comment:
```suggestion
More informations about the IO can be found on the [Digital Transformation Team website](https://teamdigitale.governo.it/en/projects/digital-citizenship.htm).
```

Reply:
It seems the weblinks brings to an English webpage automatically translated:
> This section was translated by Artificial Intelligence. [Help us improving it on GitHub](https://github.com/teamdigitale/teamdigitale.governo.it/tree/master/en/projects)

I wonder: considering that automated translations are not always accurate, is it good enough to open such a page or could it be better to open the Italian version and then let the user change the language as needed?